python/cudf/cudf/core/multiindex.py

```diff
@@ -941,9 +941,70 @@ def is_monotonic_decreasing(self):
     def argsort(self, ascending=True, **kwargs):
         return self._source_data.argsort(ascending=ascending, **kwargs)
 
+    def fillna(self, value):
```

Comment:
We now have `Series.fillna`, `DataFrame.fillna`, `Index.fillna` and `MultiIndex.fillna` -- looks like a good time to push the implementation down to the `Frame` level?

Reply:
Pushed the core implementation to `Frame` level and extending the core method in `Index` for signature difference reason.